presto-main/src/main/resources/webapp/assets/query-list.js

```diff
@@ -78,6 +78,9 @@
                 else {
                     summary = (progress == 0 ? summary : summary + " (" + progress + "%" + ")");
                 }
+                if (query.memoryPool == "reserved") {
```

Comment:
It sounded as "mark query running in reserved pool". We considered marking with color. Later with text. I think it good to put text unconditionally. Pure visually, it wouldn't be highlighted.